docs/appendix/changelog.md

```diff
@@ -10,6 +10,242 @@ aliases:
 # SME:
 ---
 
+## October 2023
+
+### Added: Test an ML model in browser
+
+Now, after you upload and train your machine learning model, you can test its results in the **Data** tab.
+With this feature you can now:
+
+- Select an image in the **Data** tab and access the **run model** section to choose and run a model
+- View the results directly above the image
+- Refine models by iteratively tagging more images for training based on observed performance
+
+To use this update, the classifier must have been trained or uploaded after September 19 2023.
+Additionally, the current version of this feature exclusively supports classification models.
```

Comment:
This is good but duplicates information that may change in the future, so I think it's best to link to the docs which should always be up to date:

```suggestion
This allows you to refine models by iteratively tagging more images for training based on observed performance.

For more information, see [Test classification models with existing images in the cloud](/services/vision/classification/#existing-images-in-the-cloud).

To use this update, the classifier must have been trained or uploaded after September 19 2023.
The current version of this feature exclusively supports classification models.
```